homeassistant/components/pterodactyl/api.py

```diff
@@ -86,23 +84,22 @@ async def async_get_data(self) -> dict[str, PterodactylData]:
 
         for identifier in self.identifiers:
             try:
-                server = await self.hass.async_add_executor_job(
-                    self.pterodactyl.client.servers.get_server,  # type: ignore[union-attr]
-                    identifier,
-                )
-                utilization = await self.hass.async_add_executor_job(
-                    self.pterodactyl.client.servers.get_server_utilization,  # type: ignore[union-attr]
-                    identifier,
+                server, utilization = await gather(
+                    self.hass.async_add_executor_job(
+                        self.pterodactyl.client.servers.get_server,  # type: ignore[union-attr]
+                        identifier,
+                    ),
+                    self.hass.async_add_executor_job(
+                        self.pterodactyl.client.servers.get_server_utilization,  # type: ignore[union-attr]
+                        identifier,
+                    ),
```

Comment:
not like this, I mean, create a `def something()` and run that in the executor

Reply:
Oops 😄 Got it now, sorry.